hpat/tests/test_hpat_jit.py

```diff
@@ -0,0 +1,142 @@
+import unittest
+import hpat
+import numba
+from hpat import *
+from numba.typed import Dict
+from collections import defaultdict
+
+class TestHpatJitIssues(unittest.TestCase):
+
+    @unittest.skip("Dict is not supported as class member")
+    def test_class_with_dict(self):
+        @jitclass([('d', Dict)])
```

Comment:
This is failing because `Dict` is not treated as a type.  It needs to be spelled as `@jitclass([('d', numba.types.DictType(types.intp, types.float64))])`.  The dictionary type has to be fully specialized. See example in https://gist.github.com/sklam/945c3304c4b132127df377fa317f5ae5.